CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Changed
+
+- Max PoV size 10mb for Kusama Asset Hub and People chains ([polkadot-fellows/runtimes/pull/718](https://github.com/polkadot-fellows/runtimes/pull/718))
```

Comment:
Why only these chains?

Reply:
update for all

Reply:
reverted it for now - some bridge tests are failing. I tried fixing it, but they pull constants from the SDK for assertions, not sure why. I asked Branislav; if there’s a simple fix, I’ll handle it. I’m trying to keep my focus on the AHM migration, so I’ve limited the changes to what’s directly related. It might look like just changing a constant, but it often brings additional complexity.